app/api/events.py

```diff
@@ -33,16 +35,36 @@ class Meta:
         self_view_many = 'v1.event_list'
         inflect = dasherize
 
+    @validates_schema
+    def validate_date(self, data):
+        if data['starts_at'] >= data['ends_at']:
+            raise UnprocessableEntity({'pointer': 'ends_at'}, "ends_at should be after starts_at")
+
+    @validates_schema
+    def validate_timezone(self, data):
+        if 'timezone' in data['timezone']:
```

Comment:
@shubham-padia 
should have been `if 'timezone' in data`
Done changes in [PR#3857](https://github.com/fossasia/open-event-orga-server/pull/3857)